test/e2e/apps/disruption.go

```diff
@@ -335,3 +364,38 @@ func createReplicaSetOrDie(cs kubernetes.Interface, ns string, size int32, exclu
 	_, err := cs.AppsV1().ReplicaSets(ns).Create(rs)
 	framework.ExpectNoError(err, "Creating replica set %q in namespace %q", rs.Name, ns)
 }
+
+func locateARunningPod(cs kubernetes.Interface, ns string) (pod *v1.Pod, err error) {
+	By("locating a running pod")
+	err = wait.PollImmediate(framework.Poll, schedulingTimeout, func() (bool, error) {
+		podList, err := cs.CoreV1().Pods(ns).List(metav1.ListOptions{})
+		if err != nil {
+			return false, err
+		}
+
+		for i := range podList.Items {
+			if podList.Items[i].Status.Phase == v1.PodRunning {
+				pod = &podList.Items[i]
+				return true, nil
+			}
+		}
+
+		return false, nil
+	})
+	return pod, err
+}
+
+func waitForPdbToBeProcessed(cs kubernetes.Interface, ns string) {
+	By("Waiting for the pdb to be processed")
+	err := wait.PollImmediate(framework.Poll, schedulingTimeout, func() (bool, error) {
+		pdb, err := cs.PolicyV1beta1().PodDisruptionBudgets(ns).Get("foo", metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		if pdb.Status.ObservedGeneration < pdb.Generation {
+			return false, nil
+		}
+		return true, nil
+	})
+	framework.ExpectNoError(err, "Waiting for the pdb to be processed", ns)
```

Comment:
Here could be
`"Waiting for the pdb to be processed in namespace %s", ns)`

Reply:
ok will change